guides/_subsections/using-smart-triggers.md

```diff
@@ -0,0 +1,96 @@
+## [Using Smart Triggers](#using-smart-triggers)
+Cryostat 2.4 agent supports custom triggers that are based on MBean metric values. You can configure Cryostat agent to start JFR recordings dynamically when these custom trigger conditions are met.
```

Comment:
We don't need to include versioning information directly in the text here - that's one of the benefits of versioning the entire guides directory :)